docs/book/events.md

```diff
@@ -0,0 +1,44 @@
+# Events
+
+The symfony/console component allows attaching an event dispatcher instance to a console application.
+During the lifetime of a console command, the application will trigger a number of events, to which you may subscribe listeners.
+Internally, laminas/laminas-cli itself adds a listener on the `Symfony\Component\Console\ConsoleEvents::TERMINATE` event in order to provide [command chains](command-chains.md).
+
+If you wish to subscribe to any of the various symfony/console events, you will need to provide an alternate event dispatcher instance.
+You may do so by defining a `Laminas\Cli\SymfonyEventDispatcher` service in your container that resolves to a `Symfony\Component\EventDispatcher\EventDispatcherInterface` instance. (We use this instead of the more generic `Symfony\Contracts\EventDispatcher\EventDispatcherInterface` so that we can use its `addListener()` method to subscribe our own listener.)
+
+As an example, your container configuration file might look like the following for laminas-mvc applications:
+
+```php
+// in config/autoload/console.global.php
```

Comment:
_Excerpt from the upcoming documentation writer's guide:_

### Filenames

A filename must be placed before a code example because there are always two parts for this information: 

1. what kind of file is this
2. and what is the name or path.

Both informations in a code comment would be too long and misplaced.

#### Examples

> Create a file for the module configuration called `module.config.php` under `laminas-mvc-tutorial/module/Album/config/`:

> …in a factory, e.g. `src/Album/Controller/AlbumControllerFactory.php`:

> …in a module configuration file, e.g. `module/Album/config/module.config.php`: